src/applications/vaos/actions/expressCare.js

```diff
@@ -115,6 +124,62 @@ export function fetchExpressCareWindows() {
   };
 }
 
+/**
+ * Fetches request limits for all active windows and selects the first one
+ * where the user has not reached their request limit
+ */
+export function fetchRequestLimits() {
+  return async (dispatch, getState) => {
+    dispatch({
+      type: FORM_FETCH_REQUEST_LIMITS,
+    });
+
+    try {
+      const activeFacilityIds = selectActiveExpressCareWindows(
+        getState(),
+        moment(),
+      ).map(window => window.facilityId);
+
+      const promiseIds = [...activeFacilityIds];
+      const requestLimits = [];
+
+      // Temporarily limit concurrent calls to 5 at at time while we
+      // wait for a new endpoint that will accept multiple facilityIds
+      while (promiseIds.length) {
+        requestLimits.push(
+          // eslint-disable-next-line no-await-in-loop
+          await Promise.all(
+            promiseIds
+              .splice(0, 5)
```

Comment:
I think we can use `slice` and not have to make a shallow copy of the activeFacilityIds array